### ISSUE TITLE
Labels are incorrectly highlighten when there is no semi

### ISSUE THREAD

**Issue by ooflorent** [gandm/language-babel#409]
<img width="164" alt="semi_label" src="https://user-images.githubusercontent.com/168498/29918072-521c832a-8e44-11e7-94d1-b2659ba09d37.png">

In the above example, `x` is the second function should also be colored in green.

**Comment by gandm** [comment]
Yeah, actually both are incorrect -I don't handle labels at all! I've had this in the back of my mind to fix but to be frank it's a bit of a pain to do.

**Comment by gandm** [comment]
@ooflorent If you get a change can you try the latest master in this repo. 
```js
// It should work for labels 
label: for() {}
label: while {}
label: do {}
label:
{
   statements
}
// BUT NOT
label: { // ambiguous as this could be a flow def of an object
   statements 
}
```

**Comment by gandm** [comment]
This change is in 2.74.0

**Comment by ooflorent** [comment]
It works like a charm! Thanks

**Comment by gandm** [comment]
Just saw an issue with break|continue labelname that is fixed in 2.74.1
